packages/lambda-powertools-pattern-basic/supplement-csv.js

```diff
@@ -0,0 +1,23 @@
+const csvToKeyValuesArray = csv =>
+  csv.split(',').filter(x => x).map(pair => pair.split(':'))
```

Comment:
the second `split` need to be a bit smarter and deal with cases like `env:stage:test` (datadog't [official doc](https://docs.datadoghq.com/tagging/) mentions this example too)

![image](https://user-images.githubusercontent.com/546969/60056226-1ae79e00-96d8-11e9-8a9e-ea8b0f8b4f62.png)